presidio-analyzer/tests/test_analyzer_engine.py

```diff
@@ -234,5 +234,22 @@ def test_when_allFields_is_true_return_all_fields(self):
         request.text = " Credit card: 4095-2609-9393-4932,  my phone is 425 8829090 " \
             "Domain: microsoft.com"
         response = analyze_engine.Apply(request, None)
+        returned_entities = [field.field.name for field in response.analyzeResults]
```

Comment:
Can you add one test when you specify all fields and specify ONE entity type and verify you get an error

Reply:
sure